pkgs/development/tools/poac/default.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, git
+, cacert
+, boost179
+, fmt_8
+, libarchive
+, libgit2
+, llvmPackages_14
+, lz4
+, openssl
+, spdlog
+}:
+
+stdenv.mkDerivation rec {
+  pname = "poac";
+  version = "0.3.7";
+
+  src = fetchFromGitHub {
+    owner = "poacpm";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-mrEtLyVnv6jggQICie210TZiG20eAacbjVXFlZqleVE=";
+  };
+
+  cmakeFlags = [
+    "-DCPM_USE_LOCAL_PACKAGES=ON"
+    "-DBOOST_ASIO_DISABLE_STD_ALIGNED_ALLOC=1" # https://github.com/msys2/MINGW-packages/issues/9827#issuecomment-947985996
+    "-DBoost_NO_BOOST_CMAKE=ON" # https://github.com/msys2/MINGW-packages/issues/5233#issuecomment-485244682
+  ];
+
+  nativeBuildInputs = [ cmake git cacert llvmPackages_14.clang ];
```

Is clang used as the C compiler? 
If this is the case, you should not put it here, but "implicitly" at `stdenv`.

```suggestion
  nativeBuildInputs = [ cmake git cacert ];
```